www/xmlResult.php

```diff
@@ -88,6 +88,8 @@
                 echo "<latency>{$test['testinfo']['latency']}</latency>\n";
                 echo "<plr>{$test['testinfo']['plr']}</plr>\n";
             }
+            if( @strlen($test['testinfo']['mobile']) )
```

Comment:
I'm fine with that.  So make it:

```
            if( isset($test['testinfo']['mobile']) )
                echo "<mobile>" . xml_entities($test['testinfo']['mobile']) .   "</mobile>\n";
```

Works fine in my tests.

I think I may have messed up this pull request though.